feature-branch-repo-management-with-cli-example/README.md

```diff
@@ -0,0 +1,3 @@
+# Automate Repository Management with the JFrog CLI
+
```

Comment:
I suggest adding a very brief introduction paragraph, describing this example. See my suggestion below. Let's also make sure that the blog indeed includes all the info needed to make this example work.

This example demonstrates how you can use GitHub Actions to automate the creation and deletion of Artifactory repositories, which correspond to feature branches that are created on GitHub.
The full description of how to set up and run this example are available on ---this blog post---.

Reply:
Done. I'll add the link for the blog post after it will be published.